aten/src/ATen/native/native_functions.yaml

```diff
@@ -2861,6 +2861,24 @@
 - func: logsumexp.names_out(Tensor self, Dimname[1] dim, bool keepdim=False, *, Tensor(a!) out) -> Tensor(a!)
   device_check: NoCheck   # TensorIterator
 
+- func: _logsumexp2(Tensor self, int[1] dim, bool keepdim=False, Tensor? b=None, bool return_sign=False) -> (Tensor out, Tensor sign_out)
+  device_check: NoCheck   # TensorIterator
+  variants: function
+  dispatch:
+    CompositeExplicitAutograd: _logsumexp2
+
+- func: _logsumexp2.out(Tensor self, int[1] dim, bool keepdim=False, Tensor? b=None, bool return_sign=False, *, Tensor(a!) out, Tensor(s!) sign_out) -> (Tensor(a!), Tensor(s!))
+  device_check: NoCheck   # TensorIterator
+  dispatch:
+    CompositeExplicitAutograd: _logsumexp2_out
+
+- func: _logsumexp2.names(Tensor self, Dimname[1] dim, bool keepdim=False, Tensor? b=None, bool return_sign=False) -> (Tensor out, Tensor sign_out)
```

Comment:
I wouldn't bother supporting named tensors unless you're very keen on them

Reply:
For simplicity I would just update the special.logsumexp schemas and the corresponding torch.logsumexp schemas (the special version of the operator doesn't support named tensors)